lib/liquid/ast.ex

```diff
@@ -0,0 +1,45 @@
+defmodule Liquid.Ast do
```

Comment:
Can you add module documentation?

Reply:
Sure, we'll add documentation in the next commits